add sponsor page for ep2024 (#523)
EuroPython/website: src/content/pages/sponsor.mdx

---
title: Sponsor
subtitle: Sponsoring EuroPython guarantees you highly targeted visibility and the opportunity to present yourself and your company to one of the largest and most diverse Python communities in Europe and beyond.
---

# Sponsor

<font size="+2.5">Support the Largest European Python Conference!</font>

Sponsoring EuroPython guarantees you highly targeted visibility and the opportunity to present yourself and your company to one of the largest and most diverse Python communities in Europe and beyond.

> Become a sponsor of EuroPython 2024. Join us this July in the **[Prague Congress Centre (PCC)](https://www.praguecc.cz/en/homepage)** and directly help promote the work of the great open-source community and further its development.

Reviewer: ```suggestion
Become a sponsor of EuroPython 2024. Join us from July 8th to 14th in the **[Prague Congress Centre (PCC)](https://www.praguecc.cz/en/homepage)** and directly help promote the remarkable open-source community's work and further its development.
```

<img src="/img/hall_rotated.png" width="3279" height="1785" />

<ButtonWithTitle title="Ready to become a sponsor and join us in Prague?" text="Sign Up Now!" href="https://docs.google.com/forms/d/1Ut6b5KaYtj1hAivVxMdaSfeNCUCNTrTq98o5v9uNg6I/" />

## 2023 Stats & Demographics

EuroPython 2023 was a hybrid conference, both in Prague & Online, combining the good of both worlds.

<div style={{textAlign: "center"}}>

> <ButtonLink href="https://drive.google.com/file/d/19kjdiSNDpK1COz2qYgJKMGUfa5Wzft5c/view?usp=sharing">2023 Stats & Demographics (PDF)</ButtonLink>

Reviewer: ```suggestion
<ButtonLink href="https://drive.google.com/file/d/19kjdiSNDpK1COz2qYgJKMGUfa5Wzft5c/view?usp=sharing">EuroPython2023 Stats & Demographics (PDF)</ButtonLink>
```

</div>

---
 # Sponsor Tiers

> At EuroPython 2024, we offer seven main packages, aiming to keep EuroPython accessible to most organisations keen to help the community.

Reviewer: ```suggestion
At EuroPython 2024, we offer eight main packages to keep EuroPython accessible to most organisations keen to help the community.
```

<div style={{textAlign: "left", marginBottom: "30px"}}>

> <ButtonLink href="https://drive.google.com/file/d/1GZxkh04YFMJaWAswK_VXwdqmS7zAd2LC/view?usp=sharing">Sponsor Packages & Options Table (PDF)</ButtonLink>

Reviewer: ```suggestion
<ButtonLink href="https://drive.google.com/file/d/10Ewe50zpmQZ0vwvgApL8l55jVDf6q-8L/view?usp=sharing">Sponsor Packages & Options (PDF)</ButtonLink>
```

&nbsp;
<ButtonLink href="https://forms.gle/ieK9LwdoRyDU4Z9a7">Sign up now!</ButtonLink>
</div>

<SponsorTiers />

> ## Optional Extras!

Reviewer: Can we plug in the description of the "Team5 + Patron", "Team10 + Patron", and "Team15 + Bronze" somehow? I don't know how exactly, but we could as ChatGPT 🤓

In addition to our main packages, we offer these add-ons as optional extras or
stand-alone sponsor options. Below are some examples.

> Please note: each add-on has limited slots and some are exclusive. Please check the full list in the PDF for details.

Reviewer: ```suggestion
Please note: each add-on has limited slots, and some are exclusive. Please check the complete list in the PDF for details.
```

- Sponsored talk on a conference day: €3,500
- Sponsored Workshop on a Workshop Day: €4,500
- Participation in recruiting session: €1,000
- Childcare Sponsors: €2,000
- Financial Aid Sponsors: €5,000
- PyLadies Lunch Sponsors: €2,000
- Open Space Room Sponsors: €2,000
- And many more!

<div style={{textAlign: "left", right: 30}}>

<ButtonLink href="https://drive.google.com/file/d/1GZxkh04YFMJaWAswK_VXwdqmS7zAd2LC/view?usp=sharing">Full list of Optional Extras (PDF)</ButtonLink>
&nbsp;
<ButtonLink href="https://forms.gle/ieK9LwdoRyDU4Z9a7">Sign up now!</ButtonLink>
</div>

## Discounts

### Early Bird

>  **10% discount** applies to launch sponsors who sign up by **15th of April**. Reach out to us to at **[sponsoring@europython.eu](mailto:sponsoring@europython.eu)**.

Reviewer: ```suggestion
 **10% discount** applies to launch sponsors who sign up by **15th of April, 2024**. Contact us at **[sponsoring@europython.eu](mailto:sponsoring@europython.eu)**.
```

### SME

> We have special offers for SMEs (small to medium sized companies) active in the

Reviewer: ```suggestion
We have special offers for SMEs (small to medium-sized companies) active in the
```
